src/db.js

```diff
@@ -26,7 +26,7 @@ const sequelize = new Sequelize({
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || 'postgres',
```

Comment:
The default password is set to 'postgres'. For security reasons, it's recommended to use a more secure password, even in development environments. Ensure that the actual password is set in the environment variables in production.